### PR TITLE
use docker testcontainer for integration tests

### PR DESCRIPTION
**Pull request by lalottacontinua**

### CODE REVIEW

File: src/main/scala/com/ubirch/util/elasticsearch/EsBulkClient.scala [hunk 1]
```diff
@@ -18,7 +18,7 @@ object EsBulkClient extends EsBulkClientBase
 
 trait EsBulkClientBase extends StrictLogging {
 
-  private val esClient: RestHighLevelClient = EsHighLevelClient.client
+  private[elasticsearch] val esClient: RestHighLevelClient = EsHighLevelClient.client
```
Comment (Hiroki6) [inline src/main/scala/com/ubirch/util/elasticsearch/EsBulkClient.scala:22]: small thing, but I feel `protected` is proper in this case, not `private`

**Review by Hiroki6 — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
     <groupId>com.ubirch.util</groupId>
     <artifactId>ubirch-elasticsearch-utils_${scala.compat.version}</artifactId>
-    <version>0.2.8-SNAPSHOT</version>
+    <version>0.2.9-SNAPSHOT</version>
```
Comment (Hiroki6) [inline pom.xml:25]: I think it's nice to have a release tag with this version and a description of this change

File: pom.xml [hunk 3]
```diff
@@ -71,19 +71,20 @@
         <timestamp>${maven.build.timestamp}</timestamp>
         <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
 
-        <scala.version>2.12.8</scala.version>
-        <scala.compat.version>2.12</scala.compat.version>
+        <scala.version>2.11.12</scala.version>
```
Comment (Hiroki6) [inline pom.xml:74]: sad..

File: pom.xml [hunk 4]
```diff
@@ -361,6 +397,33 @@
                     <skip>false</skip>
                 </configuration>
             </plugin>
+
+            <!-- code formatting -->
+            <plugin>
+                <groupId>org.antipathy</groupId>
+                <!-- The scala binary here doesn't need to match the project version -->
+                <artifactId>mvn-scalafmt_${scala.compat.version}</artifactId>
```
Comment (Hiroki6) [inline pom.xml:408]: nice

**Review by Hiroki6 — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (lalottacontinua) [reply pom.xml:25]: you mean to create a release with github as we did with the certifyAPI?

File: pom.xml (hunk 3 above)
Comment (lalottacontinua) [reply pom.xml:74]: true.

**Pull request merged**